index.js

```diff
@@ -4,65 +4,70 @@
 const path = require('path');
 const Funnel = require('broccoli-funnel');
 const mergeTrees = require('broccoli-merge-trees');
+const map = require('broccoli-stew').map;
 
-function isFastboot() {
-  return process.env.EMBER_CLI_FASTBOOT === 'true';
-}
 
 module.exports = {
   name: 'ember-medium-editor',
 
-  included(parent) {
-    this._super.included.apply(this, arguments);
-
-    // exit from hook in the fastboot mode
-    if (isFastboot()) { return; }
-
-    let options = this.getConfig(parent);
+  included(app) {
+    if (typeof app.import !== 'function' && app.app) {
+      app = app.app;
+    }
+    this.app = app;
+    this.addonConfig = this.getConfig();
 
-    let vendorPath = this.treePaths.vendor;
-    let jsPath = path.join(vendorPath, 'medium-editor', 'js');
-    let stylesPath = path.join(vendorPath, 'medium-editor', 'css');
+    // Setup paths
+    const vendorPath = this.treePaths.vendor;
```

Comment:
@willviles I prefer to use `let` inside of functions according to this [rule](https://github.com/DockYard/eslint-plugin-ember-suave/blob/master/docs/rules/no-const-outside-module-scope.md). Any advantages for using const?